src/types/Account/methods/createTransaction.spec.js

```diff
@@ -39,6 +40,10 @@ describe('Account - createTransaction', () => {
     expect(() => createTransaction.call(self, mockOpts2)).to.throw(expectedException2);
     expect(() => createTransaction.call(self, mockOpts3)).to.throw(expectedException3);
   });
+  if(process.browser){
```

Comment:
Let's add this check to particular tests and use `this.skip` with that comment.

Reply:
Similar than previous one

Reply:
https://github.com/dashevo/wallet-lib/pull/121#discussion_r431639536